docs/src/main/paradox/docs/delta/api/current/views/elasticsearch-view-api.md

```diff
@@ -94,30 +105,220 @@ include the resource metadata fields.
 
 The following sections describe the endpoints that are specific to an ElasticSearchView.
 
-The general view endpoints are described on the @ref:[parent page](index.md#endpoints).
+### Create using POST
 
-### Search Documents
+```
+POST /v1/views/{org_label}/{project_label}
+  {...}
+```
+
+The json payload:
+
+- If the `@id` value is found on the payload, this @id will be used.
+- If the `@id` value is not found on the payload, an @id will be generated as follows: `base:{UUID}`. The `base` is the
+  `prefix` defined on the resource's project (`{project_label}`).
+
+**Example**
+
+Request
+:   @@snip [create.sh](../assets/views/elasticsearch/create.sh)
+
+Payload
+:   @@snip [payload.json](../assets/views/elasticsearch/payload.json)
+
+Response
+:   @@snip [created.json](../assets/views/elasticsearch/created.json)
+
+### Create using PUT
+
+This alternative endpoint to create a view is useful in case the json payload does not contain an `@id` but you want to
+specify one. The @id will be specified in the last segment of the endpoint URI.
+
+```
+PUT /v1/views/{org_label}/{project_label}/{schema_id}/{view_id}
+  {...}
+```
+
+Note that if the payload contains an @id different from the `{view_id}`, the request will fail.
+
+**Example**
+
+Request
+:   @@snip [create-put.sh](../assets/views/elasticsearch/create-put.sh)
+
+Payload
+:   @@snip [payload.json](../assets/views/elasticsearch/payload.json)
+
+Response
+:   @@snip [created.json](../assets/views/elasticsearch/created-put.json)
+
+### Update
+
+This operation overrides the payload.
+
+In order to ensure a client does not perform any changes to a resource without having had seen the previous revision of
+the view, the last revision needs to be passed as a query parameter.
+
+```
+PUT /v1/views/{org_label}/{project_label}/{schema_id}/{view_id}?rev={previous_rev}
+  {...}
+```
+
+... where `{previous_rev}` is the last known revision number for the view.
+
+@@@ note { .warning }
+
+Updating a view creates a new Elasticsearch index and deletes the existing one. The indexing process will start from the
+beginning.
+
+@@@
+
+**Example**
+
+Request
+:   @@snip [update.sh](../assets/views/elasticsearch/update.sh)
+
+Payload
+:   @@snip [payload.json](../assets/views/elasticsearch/payload.json)
+
+Response
+:   @@snip [updated.json](../assets/views/elasticsearch/updated.json)
+
+## Tag
+
+Links a view revision to a specific name.
+
+Tagging a view is considered to be an update as well.
+
+```
+POST /v1/views/{org_label}/{project_label}/{schema_id}/{view_id}/tags?rev={previous_rev}
+  {
+    "tag": "{name}",
+    "rev": {rev}
+  }
+```
+
+... where
+
+- `{previous_rev}`: is the last known revision number for the resource.
+- `{name}`: String - label given to the view at specific revision.
+- `{rev}`: Number - the revision to link the provided `{name}`.
+
+**Example**
+
+Request
+:   @@snip [tag.sh](../assets/views/elasticsearch/tag.sh)
```

Comment:
this file is empty according to this PR

Reply:
And what the picture shows is a wrong curl request too